docs/Documentation/Applications/vasp.md

```diff
@@ -0,0 +1,537 @@
+    #SBATCH --account=myaccount
+
+    module purge
+    ml gcc
+    ml vasp/6.1.1-openmpi
+
+    # some extra lines that have been shown to improve VASP reliability on Vermilion
+    ulimit -s unlimited
+    export UCX_TLS=tcp,self
+    export OMP_NUM_THREADS=1
+    ml ucx
+
+    # lines to set "ens7" as the interconnect network
+    module use /nopt/nrel/apps/220525b/level01/modules/lmod/linux-rocky8-x86_64/gcc/12.1.0
+    module load openmpi
+    OMPI_MCA_param="btl_tcp_if_include ens7"
+
+    srun --mpi=pmi2 -n 60 vasp_std
+    ```
+??? example "Vermilion: VASP 5 (Intel MPI) on CPUs"
+  
+    ```
+    #!/bin/bash
+    #SBATCH --job-name=vasp
+    #SBATCH --nodes=1
+    #SBATCH --time=8:00:00
+    ##SBATCH --error=std.err
+    ##SBATCH --output=std.out
+    #SBATCH --partition=lg
+    #SBATCH --exclusive
+    #SBATCH --account=myaccount
+
+    module purge
+
+    ml vasp/5.4.4
+
+    source /nopt/nrel/apps/220525b/myenv.2110041605
+    ml intel-oneapi-compilers/2022.1.0-k4dysra
+    ml intel-oneapi-mkl/2022.1.0-akthm3n
+    ml intel-oneapi-mpi/2021.6.0-ghyk7n2
+
+    # some extra lines that have been shown to improve VASP reliability on Vermilion
+    ulimit -s unlimited
+    export UCX_TLS=tcp,self
+    export OMP_NUM_THREADS=1
+    ml ucx
+
+    srun --mpi=pmi2 -n 60 vasp_std
+
+    # If the multi-node calculations are breaking, replace the srun line with this line
+    # I_MPI_OFI_PROVIDER=tcp mpirun -iface ens7 -np 60 vasp_std
+    ```
+??? example "Vermilion: VASP 6 (OpenACC) on GPUs"
+  
+    ```
+    #!/bin/bash
+    #SBATCH --job-name=vasp
+    #SBATCH --nodes=2
+    #SBATCH --time=1:00:00
+    ##SBATCH --error=std.err
+    ##SBATCH --output=std.out
+    #SBATCH --partition=gpu
+    #SBATCH --gpu-bind=map_gpu:0,1,0,1
+    #SBATCH --exclusive
+    #SBATCH --account=myaccount
+
+    # Load the OpenACC build of VASP
+    ml vasp/6.3.1-nvhpc_acc
+
+    # Load some additional modules
+    module use  /nopt/nrel/apps/220421a/modules/lmod/linux-rocky8-x86_64/gcc/11.3.0/
+    ml nvhpc
+    ml fftw
+
+    mpirun -npernode 1 vasp_std > vasp.$SLURM_JOB_ID
+    ```
+
+!!! Note
+    On Vermilion, VASP runs more performantly on a single node. Many issues have been reported for running VASP on multiple nodes, especially when requesting all available cores in each node. In some cases, VASP runtimes on 2 nodes have been observed to be double (or more) the run times on a single node. In order for MPI to work reliably on Vermilion, it is necessary to specify the interconnect network that Vermilion should use to communicate between nodes. This is documented in each of the scripts below. Different solutions exist for Open MPI and Intel MPI. The documented recommendations for setting the interconnect network have been shown to work well for multi-node jobs on 2 nodes, but aren't guaranteed to produce succesful multi-node runs on 4 nodes. The Open MPI multi-node jobs are more reliable on Vermilion, but Intel MPI VASP jobs show better runtime performance. 
```

Comment:
The content of this note is also repeated in the Performance Recommendations, do you think we could remove this, or maybe just draw attention to the below section in this note?

Reply:
I do think this needs to be near the run script so people are made aware of the issues in advance and we can reduce the number of tickets that come through about this.